pipelined/bevy_render2/src/options.rs

```diff
@@ -0,0 +1,218 @@
+use std::borrow::Cow;
```

Comment:
I'm thinking we should just use the wgpu Features and Limits types now instead of maintaining our own mappings. Same goes for wgpu Backends

Reply:
Ah same goes for PowerPreference

Reply:
Ok great. I'll change it that way. Better indeed. I actually made it like that initially, but then backported all the types too :)

Reply:
Awesome!

Reply:
Just made the changes. I also re-exported the relevant wgpu structs under render2::options so users can easily use them. Let me know if you want any additional changes.